README.md

```diff
@@ -161,6 +161,10 @@ The packages in this repo provide two options: a [command](#trace-command) and a
 > For more information, refer to the [design document](./doc/design_ros_2.md#general-guidelines).
 > The [launch file action](#launch-file-trace-action) is designed to automatically start tracing before the application launches.
 
+> [!TIP]
+> Configuring a tracing session in [snapshot mode](#tracing-in-snapshot-mode) or [dual session mode](#dual-session-tracing) stores trace data in the memory without writing to disk until demanded.
+> This can be leveraged to start recording traces after application launch without losing initialization data, eliminating the need to record trace data before application launch and preventing accumulation of unwanted trace files when not actively analyzing.
```

Comment:
I know what you mean, but the sentence is a bit repetitive. It reads like: "this can be leveraged to _avoid doing X_, _eliminating the need to do X_ and [...]."

I think this would help make it simpler and make the point/intention clearer:

```suggestion
> This can be leveraged to start recording traces as needed after application launch without losing initialization data, eliminating the need to start recording all trace data before application launch and preventing accumulation of unwanted trace files when not actively analyzing.
```